backends/server/dist/server.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
```

Comment:
Remove the dist folder, no reason to commit this.
Make sure after you delete this, you add this to a .gitignore file